Source/ACE.Server/WorldObjects/Player.cs

```diff
@@ -40,6 +40,7 @@ public partial class Player : Creature, IPlayer
 
         public bool LastContact = true;
         public bool IsJumping = false;
+        //public bool DoNotSave = false;
```

Comment:
???

Reply:
Left over from initial testing, meant to remove.